plugin-volume/alsaengine.cpp

```diff
@@ -33,6 +33,32 @@
 #include <QSocketNotifier>
 #include <QtDebug>
 
+MixerHandler::MixerHandler(snd_mixer_t * mixer, QObject * parent /*= nullptr*/)
+    : QObject{parent}
+    , m_mixer{mixer}
+{
+    if (nullptr != m_mixer)
+    {
+        // setup eventloop handling
+        struct pollfd pfd;
+        if (snd_mixer_poll_descriptors(m_mixer, &pfd, 1)) {
+            QSocketNotifier *notifier = new QSocketNotifier(pfd.fd, QSocketNotifier::Read, this);
+            connect(notifier, &QSocketNotifier::activated, this, [this] {
+                    const int err = snd_mixer_handle_events(m_mixer);
+                    if (0 > err)
+                        emit handlingError(err);
+                });
```

Comment:
Is the indentation a bit odd here?

Reply:
What indentation would you recommend?

Reply:
If it is indented as intended, I'd recommend keeping it as is; what works best for you.  :]

I think it might mainly be the closing `});` that jumped at me, which isn't aligned with any of the braces or paren, and it made me think it might have looked different in an editor (different space/tab handling etc.), and possibly not intentional, so I just wanted to point it out.